modules/exploits/unix/webapp/wp_infinitewp_auth_bypass.rb

```diff
@@ -0,0 +1,121 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HTTP::Wordpress
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'WordPress InfiniteWP Client Authentication Bypass',
+      'Description'    => %q{
+        This module exploits an authentication bypass in the WordPress
+        InfiniteWP Client plugin to log in as an administrator and execute
+        arbitrary PHP code. A valid administrator username is required.
```

Comment:
```suggestion
        arbitrary PHP code by overwriting the specified `PLUGIN_FILE`.
        A valid administrator username is required.
```

Reply:
I'll update this according to https://github.com/rapid7/metasploit-framework/pull/12853#discussion_r368242294. The docs already explain in full and will be adjusted accordingly. Also, Markdown is not supported in module descriptions yet.